### PR TITLE
[DXP Cloud] LRDOCS-9966 POC: Migration section

### PR DESCRIPTION
**Pull request by Alec-Shay**
See: https://issues.liferay.com/browse/LRDOCS-9966

This is a proof-of-concept to demonstrate what a new migration section (taking the place of a Migration Guide) for DXP Cloud could look like.

Please let me know if you have any thoughts/concerns regarding naming (too wordy, redundant?). I also (knowingly) somewhat broke convention by numbering the H1 titles in the articles, so that in the side nav, they can be seen and referenced as a list of steps (whereas I'm not sure the file names would work with the numbering, especially the colon). Feel free to independently revert/drop that commit if that is a dumb idea, but I thought it might look better and fit the "Migration Guide" idea better, too.

Also, please do not merge this into master just yet; the team wants to iteratively see progress with this and test the idea out before fully committing to it. Instead, if you are all good with this, can you please push this up to **UAT** to show as a POC? We'd then likely follow up with more iterations filling in the placeholders. Thanks!

P.S., since this is going through a kind of special workflow (in that it's not just being merged now and it could bounce back and forth a bit, plus the UAT step, etc.), I'm not sure what to do with the ticket status... especially since I made the parent ticket an epic, which _perhaps_ was not correct... But I'm not sure.

### CODE REVIEW

**Comment by jrhoun** [comment]
I went ahead and pushed to UAT earlier today, can I close this pr @Alec-Shay ?

**Comment by Alec-Shay** [comment]
Hi @jrhoun,

For the purposes of this PR specifically, I think it's okay to close. But, just to confirm, are you reviewing this yourself yet at this point? Or are you going to wait until a later stage of these iterations when the Cloud/design team (presumably) would have already had more of a hand in directing/signing off on the changes?

**Comment by jrhoun** [comment]
I figure you'd just send me a new pull for future iterations. I'll probably keep messing w/ the branch I have locally over the next day or two to test the svg issues though.

**Comment by Alec-Shay** [comment]
I plan to. And that works

**Comment by jrhoun** [comment]
Closing. Thanks.
